apps/lend/src/store/createMarketListSlice.ts

```diff
@@ -86,15 +88,18 @@ const createMarketListSlice = (set: SetState<State>, get: GetState<State>): Mark
     filterSmallMarkets: (api, owmDatas) => {
       const { chainId } = api
       const capAndAvailableMapper = get().markets.statsCapAndAvailableMapper[chainId] ?? {}
-      const usdRatesMapper = get().usdRates.tokens
       const { smallMarketAmount, marketListShowOnlyInSmallMarkets } = networks[chainId]
       return owmDatas.filter(({ owm }) => {
         const { cap } = capAndAvailableMapper[owm.id] ?? {}
-        const token = owm.borrowed_token
-        const usdRate = usdRatesMapper[token.address]
+        const usdRate = queryClient.getQueryData<number>(
```

Comment:
Wrap it with a factory function to don't use keys outside the entity
```suggestion
        const usdRate = getTokenQueryData('usdRate', { chainId, tokenAddress })
```

Reply:
this one is also not going to work, we need the tokenKeys to be able to create the key